packages/fether-react/src/Accounts/CreateAccount/AccountName/AccountName.js

```diff
@@ -98,35 +95,37 @@ class AccountName extends Component {
     const currentStep = pathname.slice(-1);
 
     return (
-      <div key='createAccount'>
+      <form key='createAccount' onSubmit={this.handleSubmit}>
         <div className='text'>
           <p>Please give this account a name:</p>
         </div>
         <FetherForm.Field
           label='Name'
           onChange={this.handleChangeName}
-          onKeyPress={this.handleKeyPress}
+          autoFocus
```

Comment:
props should be ordered alphabetically (unless there's some more meaning ordering).